partiql-planner/src/main/kotlin/org/partiql/planner/internal/ir/Nodes.kt

```diff
@@ -2,6 +2,7 @@
 
```

Comment:
Consider rebasing so we have generate internal IRs

Reply:
I am thinking: 
1. partiql-spi-planner -> partiql-spi
2. partiql-plugin-impl -> partiql-spi
3. delete all partiql-plugin-xxxx
4. branch off partiql-spi to continue development
5. partiql-spi -> main

-> means get merged into.